client/src/api/index.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
```

Comment:
Please create shared http service, that wraps all basic requests(get, post, put, delete). And than use it in api layer.

Reply:
Done